editor/animation_bezier_editor.cpp

```diff
@@ -1526,20 +1540,29 @@ void AnimationBezierTrackEdit::_zoom_callback(float p_zoom_factor, Vector2 p_ori
 	queue_redraw();
 }
 
+Array AnimationBezierTrackEdit::float_to_bezier_key_value(float p_val) {
+	Array new_point;
+	new_point.resize(5);
+
+	new_point[0] = p_val;
+	new_point[1] = -0.25;
+	new_point[2] = 0;
+	new_point[3] = 0.25;
+	new_point[4] = 0;
+
+	return new_point;
+}
+
+float AnimationBezierTrackEdit::bezier_to_float_key_value(Array p_val) {
+	return p_val[0];
+}
```

Comment:
It is a naming issue, but I think the function name does not express the entity. Wouldn't a name like `make_default_bezier_key()` and `get_bezier_key_value()` be better?

Reply:
yes, that makes sense